src/libhost1x/host1x-drm.c

```diff
@@ -587,10 +587,13 @@ static struct host1x_bo *drm_bo_import(struct host1x *host1x,
 static int drm_framebuffer_init(struct host1x *host1x,
 				struct host1x_framebuffer *fb)
 {
-	uint32_t handles[1], pitches[1], offsets[1], format;
+	uint32_t handles[4], pitches[4], offsets[4], format;
```

Comment:
Hmm, why does these need to be four times as big?

Reply:
Yes, kernel checks all 4 elements and if any of them is unused and isn't '0', IOCTL fails. I think we were just lucky by now, IOCTL fails with the use of modifier.

Reply:
OK, but this sounds like an unrelated change. If so, perhaps it deserves its own commit (and commit message)?

Reply:
Dunno, that would be very pedantic :) I'll just amend the commit message.

Reply:
That's fine also.

Reply:
Done, updated the message.